src/app/ui/resources_listbox.cpp

```diff
@@ -67,7 +68,7 @@ void ResourceListItem::onPaint(PaintEvent& ev)
     text(),
     fgcolor,
     gfx::ColorNone,
-    gfx::Point(bounds.x + 2 * guiscale(), bounds.y + bounds.h / 2 - g->font()->height() / 2));
+    gfx::Point(bounds.x + (2 * guiscale()), bounds.y + (bounds.h / 2) - (g->font()->height() / 2)));
```

Comment:
This change generates a conflict and is not needed in the PR.

Please remove everything that is superfluous from the PR and commit just the required changes to fix the issue.